### DISCUSSION TITLE
Not reporting hashrate to Ethermine pool but submitting shares works

### DISCUSSION THREAD

**Discussion by paulpoco** [RainbowMiner/RainbowMiner#1676] [Q&A] [answered]
Not reporting hashrate to Ethermine pool but submitting shares works
[debug_2021-07-02.zip](https://github.com/RainbowMiner/RainbowMiner/files/6757430/debug_2021-07-02.zip)
![Screenshot_20210702-162754_Chrome](https://user-images.githubusercontent.com/801414/124336345-8465d700-db52-11eb-902f-0348243e3541.jpg)

**Comment by RainbowMiner** [comment]
That's the Ethminer. That miner has some compatibility problems on some pools, in case the report-hashrate feature is enabled. So I leave it disabled, currently. If you like, you can enable it any time:
- edit `.\Config\miners.config.txt`
- find Ethminer and then the Ethash/Etchash algorithm
- add `-RH` (or `--report-hashrate`) to the Params record:
```
    "Params": "-RH",
```

**Comment by paulpoco** [comment]
@RainbowMiner 

I have Ethminer disabled in main config on server config and it shows in the config of the actual PC the diags are from.
![image](https://user-images.githubusercontent.com/801414/124368485-55b13480-dc16-11eb-9026-2a2aaf38132c.png)

Is a DEV round turning Ethminer on and then when DEV/Donation round is finished Ethminer is not turned off?

**Comment by RainbowMiner** [comment, marked as answer]
In fact, the dev round respects the excluded miners. But there is a second culprit: Nsfminer - this is a fork of Ethminer. This miner needs that `-RH` parameter to report the hashrate to pools, as well. Also, it has that very same compatibility problem on some pools.

**Comment by paulpoco** [comment]
Ok thanks
